packages/gamut-labs/src/experimental/GlobalHeader/GlobalHeaderItems.tsx

```diff
@@ -0,0 +1,208 @@
+  type: 'logo',
+};
+
+export const myHome: AppHeaderLinkItem = {
+  dataTestId: 'header-home',
+  id: 'my-home',
+  text: 'My Home',
+  href: '/learn',
+  trackingTarget: 'topnav_home',
+  type: 'link',
+};
+
+export const courseCatalog: AppHeaderLinkItem = {
+  dataTestId: 'header-catalog',
+  id: 'course-catalog',
+  text: 'Course Catalog',
+  href: '/catalog',
+  trackingTarget: 'topnav_catalog',
+  type: 'link',
+};
+
+export const resourcesDropdown: AppHeaderDropdownItem = {
+  id: 'resources',
+  text: 'Resources',
+  popover: [
+    {
+      id: 'cheatsheets',
+      href: '/resources/cheatsheets/all',
+      trackingTarget: 'topnav_resources_cheatsheets',
+      text: 'Cheatsheets',
+      type: 'link',
+    },
+    {
+      id: 'articles',
+      href: '/articles',
+      trackingTarget: 'topnav_resources_articles',
+      text: 'Articles',
+      type: 'link',
+    },
+    {
+      id: 'blog',
+      href: 'https://news.codecademy.com/',
+      trackingTarget: 'topnav_resources_blog',
+      text: 'Blog',
+      type: 'link',
+    },
+  ],
+  trackingTarget: 'topnav_resources',
+  type: 'dropdown',
+};
+
+export const communityDropdown: AppHeaderDropdownItem = {
+  id: 'community',
+  text: 'Community',
+  popover: [
+    {
+      id: 'forums',
+      href: 'https://discuss.codecademy.com/',
+      trackingTarget: 'topnav_community_forums',
+      text: 'Forums',
+      type: 'link',
+    },
+    {
+      id: 'chat',
+      href: 'https://discord.com/invite/codecademy',
+      newTab: true,
+      trackingTarget: 'topnav_chat',
+      text: 'Chat',
+      type: 'link',
+    },
+    {
+      id: 'chapters',
+      href: 'https://community.codecademy.com/',
+      trackingTarget: 'topnav_community_forums',
+      text: 'Chapters',
+      type: 'link',
+    },
+    {
+      id: 'events',
+      href: '/events',
+      trackingTarget: 'topnav_community_events',
+      text: 'Events',
+      type: 'link',
+    },
+  ],
+  trackingTarget: 'topnav_community',
+  type: 'dropdown',
+};
+
+export const plansPricingDropdown: AppHeaderDropdownItem = {
+  id: 'plans-pricing',
+  text: 'Plans + Pricing',
+  popover: [
+    {
+      id: 'pro-membership',
+      href: '/pricing',
+      trackingTarget: 'topnav_pro_membership',
+      text: 'Pro Membership',
+      type: 'link',
+    },
+    {
+      id: 'for-business',
+      href: '/business',
+      trackingTarget: 'topnav_pricing_business',
+      text: 'For Business',
+      type: 'link',
+    },
+    {
+      id: 'for-students',
+      href: '/student-center',
+      trackingTarget: 'topnav_pricing_business',
+      text: 'For Students',
+      type: 'link',
+    },
+  ],
+  trackingTarget: 'topnav_pricing',
+  type: 'dropdown',
+};
+
+export const forEnterprise: AppHeaderLinkItem = {
+  id: 'for-enterprise',
+  trackingTarget: 'topnav_business',
+  text: 'For Enterprise',
+  href: '/business',
+  type: 'link',
+};
+
+export const search = (
+  renderSearch: () => ReactElement
```

Comment:
`ReactNode` is a little bit more permissive, so this will be a little more friendly for consumers. Here and elsewhere.

```suggestion
  renderSearch: () => ReactNode
```